### ISSUE TITLE
Web Service Issues

### ISSUE THREAD

**Issue by p3hndrx** [Atomicorp/gvm#54]
I am getting frequent messages in the web interface:
_An error occurred during making the request. Most likely the web server does not respond._

With an occasional:
_The NVT you were looking for could not be found.
You might have followed an incorrect link and the NVT does not exist._

The page loads, but seems slower than normal.
What are my options for troubleshooting?

Using: 
```
# gsad --version
Greenbone Security Assistant Version 21.4.2
# gvmd --version
Greenbone Vulnerability Manager 21.4.3

Installed Packages
Name         : gvm
Version      : 21.4.0
Release      : 18459.el8.art
Architecture : noarch
Size         : 87 k
Source       : gvm-21.4.0-18459.el8.art.src.rpm
Repository   : @System
From repo    : atomic
Summary      : The Greenbone Vulnerability Management (GVM) suite
URL          : http://www.openvas.org
License      : AGPL
Description  : Greenbone Vulnerability Management (GVM) is a meta-package encompassing all of the components from GVM including OpenVAS.
```

**Comment by atomicturtle** [comment]
Was this on a new installation? The NVT database can take a bit to load into postgres, so maybe that is the culprit?

**Comment by atomicturtle** [comment]
closed since 21.4.3 is out now, if you're still having problems just open a new issue for that version. Thanks again!
